inputs-and-results/benchmarks/qm-benchmarks/README.md

```diff
@@ -0,0 +1,103 @@
+
+
+This directory contains the scripts needed to compute the ddE, RMSD, and TDF metrics for a set
+of force fields against a QCArchive optimization data collection such as the `OpenFF Full Optimization Benchmark 1` 
+QCArchive data set as was used in the [Lim and Hahn study](https://doi.org/10.26434/chemrxiv.12551867.v2) (see the 
+Acknowledgments section below for more details).
+
+## Running the benchmarks
+
+1) Run the `01-setup.py` script to retrieve the relevant QC data from the QCArchive and filter out
+   and incomplete or undesirable (e.g. records whereby the molecule connectivity changed during 
+   the QM minimization) records. The dataset to retrieve and benchmark against must be specified.
+   For the Lim and Hahn study:
+   
+   ```shell
+   python 01-setup.py "OpenFF Full Optimization Benchmark 1"
+   ```
+   
+   or for the Industrial partner benchmarks:
+
+   ```shell
+   python 01-setup.py "OpenFF Industry Benchmark Season 1 v1.1"
+   ```
+
+2) 
+
+    a. Run the `02-a-chunk-qm.py` script to split the `01-processed-qm.sdf` file produced by step 1) into
+       many smaller chunks. This allows the next step to be parallelized across many different CPUs / workers.
+       By default the split chuncks will be stored in a new `02-chunks` directory.
+
+    b. Perform an MM energy minimization for each of the chunked files and for each force field of interest, e.g.:
+
+   ```shell
+   python 02-b-minimize.py -ff openff-2.0.0.offxml \ 
+                           -i 02-chunks/01-processed-qm-1.sdf \
+                           -o 02-outputs/openff-2-0-0-1.sdf
+   ```
+
+3) a. Using the MM minimized structures from step 2 compute the metrics RMSD/ddE/TFD.
+
+   ```shell
+   python 03-a-compute-metrics.py --input "03-force-fields.json" --index 1 \
+               --output 03-outputs/03-metrics-1.csv
+   ```
+   b. Concatenate the metrics files in csv format generated above 
+   ```shell
+   python 03-b-join-metrics.py 03-outputs/* -o 03-metrics.csv
+   ```
+
+4) Plot the metrics by running the `04-plot-metrics.py` script, which will be stored in `04-outputs`. The directory `04-outputs/full` contains the metrics on overall set and the other directories are subsets of R-groups.
+
+# File Manifest
+
```

Comment:
The descriptions are great can we reformat this slightly to make it clear what is a script and what is the associated output is? Maybe something like
-01-setup.py - Description ...
    - list of outputs - descriptions

Reply:
I took a shot at it, please take a look when you have time.